src/components/basics/Cardinal.js

```diff
@@ -12,19 +12,19 @@ const Count = styled.div`
   color: ${color.dark};
   display: block;
 
-  ${props =>
+  ${(props) =>
```

Comment:
Did the prettier config change? Curious why this happened a bunch

Reply:
Prettier made this change a while back. Don't think it's a config thing. I use the VS code plugin. It respects the project level config.

Reply:
Oh yeah hmm. I just pulled down and got the same result. I guess we just haven't touched this file in a while ha.